gateway/service/src/main/java/org/eclipse/ditto/gateway/service/endpoints/actors/AbstractHttpRequestActor.java

```diff
@@ -346,6 +348,15 @@ private void rememberResponseLocationUri(final CommandResponse<?> commandRespons
         }
     }
 
+    private void handleCheckPermissions(final CheckPermissions command) {
+        final ActorRef checkPermissionsActor = getContext().actorOf(
+                CheckPermissionsActor.props(proxyActor, getSelf(), getReceiveTimeout(command, commandConfig)),
+                CheckPermissionsActor.ACTOR_NAME
```

Comment:
@hu-ahmed another thing which I didn't see before ..
We cannot use the static name here to create the actor.
As we would get a "Actor name conflict" when handling 2 `checkPermissions` API calls at the same time.

Instead, just start the actor without providing a name - Pekko will then create a random actor name.